qualibration_graphs/superconducting/calibrations/13b_chevron_11-02_1ns.py

```diff
@@ -0,0 +1,367 @@
+# %% {Imports}
+import warnings
+from dataclasses import asdict, dataclass
+from typing import List, Literal, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import xarray as xr
+from calibration_utils.chevron_cz import (
+    Parameters,
+    fit_raw_data,
+    log_fitted_results,
+    plot_raw_data_with_fit,
+    process_raw_dataset,
+)
+from qm import SimulationConfig
+from qm.qua import *
+from qualang_tools.bakery import baking
+from qualang_tools.loops import from_array
+from qualang_tools.multi_user import qm_session
+from qualang_tools.results import fetching_tool, progress_counter
+from qualang_tools.units import unit
+from qualibrate import NodeParameters, QualibrationNode
+from qualibration_libs.data import XarrayDataFetcher
+from qualibration_libs.parameters import get_qubit_pairs, get_qubits
+from qualibration_libs.runtime import simulate_and_plot
+from quam_builder.architecture.superconducting.custom_gates.cz import CZGate
+from quam_config import Quam
+from scipy.optimize import curve_fit
+
+from quam.components.pulses import SquarePulse
+
+# %% {Node_parameters}
+description = """
+Unipolar CPhase Gate Calibration
+This sequence measures the time and detuning required for a unipolar CPhase gate. The process involves:
+
+1. Preparing both qubits in their excited states.
+2. Applying a flux pulse with varying amplitude and duration.
+3. Measuring the resulting state populations as a function of these parameters.
+4. Fitting the results to a Ramsey-Chevron pattern.
+
+From this pattern, we extract:
+- The coupling strength (J2) between the qubits.
+- The optimal gate parameters (amplitude and duration) for the CPhase gate.
+
+The Ramsey-Chevron pattern emerges due to the interplay between the qubit-qubit coupling and the flux-induced detuning,
+allowing us to precisely calibrate the CPhase gate.
+
+Prerequisites:
+- Calibrated single-qubit gates for both qubits in the pair.
+- Calibrated readout for both qubits.
+- Initial estimate of the flux pulse amplitude range.
+
+Outcomes:
```

Comment:
Need to say that if the macros called `"cz_unipolar"` doesn't exist, then it will be created and added to the state